src/test/java/org/folio/logintest/UserMock.java

```diff
@@ -1,17 +1,21 @@
 package org.folio.logintest;
 
+import static java.lang.Thread.sleep;
+import static org.folio.util.LoginAttemptsHelper.LOGIN_ATTEMPTS_CODE;
+import static org.folio.util.LoginAttemptsHelper.LOGIN_ATTEMPTS_TIMEOUT_CODE;
+
+import java.util.UUID;
+
+import org.drools.core.spi.Accumulator.SafeAccumulator;
```

Comment:
Please remove this unused import (which will not work with RMB 31 where Drools is removed)